src/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Suspense } from 'react'
 import ReactDOM from 'react-dom'
-import 'core-js/es/object'
+// import 'core-js/es/object'
```

Comment:
Please remove this if you no longer need it rather than commenting out

Reply:
this has been fixed in another PR